Plugin/src/gub_graphics.c

```diff
@@ -657,7 +657,9 @@ static void gub_destroy_graphic_context_egl(GUBGraphicContextEGL *gcontext)
 
 static const gchar *gub_get_video_branch_description_egl()
 {
-    return "glupload ! glcolorconvert ! video/x-raw(memory:GLMemory),texture-target=2D ! fakesink sync=1 qos=1 name=sink";
+	return "glupload ! glcolorconvert ! video/x-raw(memory:GLMemory),texture-target=2D ! fakesink sync=1 qos=1 name=sink";
```

Comment:
return "glupload ! glcolorconvert ! video/x-raw(memory:GLMemory),texture-target=2D, format=(string)RGBA ! fakesink sync=1 qos=1 name=sink";

for all colors